cfme/tests/cli/test_appliance_update.py

```diff
@@ -98,49 +62,133 @@ def test_update_yum(appliance_preupdate, appliance):
-        assert appliance.version == update_embedded_appliance.version
-
-    wait_for(is_appliance_updated, func_args=[update_embedded_appliance], num_sec=900)
-    assert wait_for(func=lambda: update_embedded_appliance.is_embedded_ansible_running, num_sec=30)
-    assert wait_for(func=lambda: update_embedded_appliance.is_rabbitmq_running, num_sec=30)
-    assert wait_for(func=lambda: update_embedded_appliance.is_nginx_running, num_sec=30)
-    assert update_embedded_appliance.ssh_client.run_command(
-        'curl -kL https://localhost/ansibleapi | grep "Ansible Tower REST API"')
+    update_appliance(enabled_embedded_appliance)
+    wait_for(do_appliance_versions_match, func_args=(appliance, enabled_embedded_appliance),
+             num_sec=900, delay=20, handle_exception=True)
+    assert wait_for(func=lambda: enabled_embedded_appliance.is_embedded_ansible_running, num_sec=90)
+    assert wait_for(func=lambda: enabled_embedded_appliance.is_rabbitmq_running, num_sec=60)
+    assert wait_for(func=lambda: enabled_embedded_appliance.is_nginx_running, num_sec=60)
+    assert enabled_embedded_appliance.ssh_client.run_command(
+        'curl -kL https://localhost/ansibleapi | grep "AWX REST API"')
+    repositories = enabled_embedded_appliance.collections.ansible_repositories
+    name = "example_{}".format(fauxfactory.gen_alpha())
+    description = "edited_{}".format(fauxfactory.gen_alpha())
+    repository = repositories.create(
+        name,
+        REPOSITORIES[0],
+        description=description)
+    view = navigate_to(repository, "Details")
+    refresh = view.toolbar.refresh.click
+    wait_for(
+        lambda: view.entities.summary("Properties").get_text_of("Status") == "successful",
+        timeout=60,
+        fail_func=refresh
+    )
+
+
+@pytest.mark.ignore_stream("upstream")
+def test_update_distributed_webui(ext_appliances_with_providers, appliance, request, old_version):
+    """ Tests updating an appliance with providers, also confirms that the
+            provisioning continues to function correctly after the update has completed"""
+    update_appliance(ext_appliances_with_providers[0])
+    update_appliance(ext_appliances_with_providers[1])
+    wait_for(do_appliance_versions_match, func_args=(appliance, ext_appliances_with_providers[0]),
+             num_sec=900, delay=20, handle_exception=True)
+    wait_for(do_appliance_versions_match, func_args=(appliance, ext_appliances_with_providers[1]),
+             num_sec=900, delay=20, handle_exception=True)
+    # Verify that existing provider can detect new VMs on both apps
+    virtual_crud_appl1 = provider_app_crud(VMwareProvider, ext_appliances_with_providers[0])
+    virtual_crud_appl2 = provider_app_crud(VMwareProvider, ext_appliances_with_providers[1])
+    vm1 = provision_vm(request, virtual_crud_appl1)
+    vm2 = provision_vm(request, virtual_crud_appl2)
+    assert vm1.provider.mgmt.is_vm_running(vm1.name), "vm is not running"
```

Comment:
Required: use soft_assert fixture if you need to check vm2 is running, even if vm1 failed